docs/pages/includes/config-reference/app-service.yaml

```
@@ -22,7 +22,8 @@ app_service:
       # Optional: For access to cloud provider APIs, specify the cloud
       # provider. Allowed values are "AWS", "Azure", and "GCP".
       cloud: ""
-      # URI and Port of Application.
+      # URI and Port of Application. For TCP applications
```

If I'm being pedantic, port is part of a URI..
```suggestion
      # URI of Application. For TCP applications
```

---

thanks @zmb3 , applied!